cookbook/Chapter 8 - How to deal with timestamps.py

```diff
@@ -1,50 +1,106 @@
 # %%
 import pandas as pd
+import polars as pl
 
 # %%
 # Parsing Unix timestamps
 # It's not obvious how to deal with Unix timestamps in pandas -- it took me quite a while to figure this out. The file we're using here is a popularity-contest file of packages.
 
 # Read it, and remove the last row
-popcon = pd.read_csv(
-    "../data/popularity-contest",
-    sep=" ",
-)[:-1]
-popcon.columns = ["atime", "ctime", "package-name", "mru-program", "tag"]
-popcon[:5]
+# Step 1: Read the text file line by line (plain text handling)
+file_path = r"C:/Users/psingh/polars_update/data/popularity-contest"
```

Comment:
File path contains username that is local and will not be able to run on other pcs. Consider use relative path.